pkg/util/command_util.go

```diff
@@ -38,8 +38,7 @@ import (
 
 // for testing
 var (
-	getIdsFromUsernameAndGroup = getIdsFromUsernameAndGroupImpl
-	getGroupFromStr            = getGroup
+	GetUIDAndGID = getUIDAndGID
```

Comment:
```suggestion
	getUIDAndGIDFunc = getUIDAndGID
```

I'd rather not export this method if we can avoid it.